Tools.Test.Database/Model/Tasks/CreateKitosUserTask.cs

```diff
@@ -84,6 +86,12 @@ private void AssignOrganizationRole(KitosContext context, User newUser, Organiza
 
             context.OrganizationRights.AddOrUpdate(x => x.UserId, newRight);
             context.SaveChanges();
+
+            if (_multiOrganization)
```

Comment:
Lad os i stedet tage en komma-sepereret liste af organisationer som brugeren skal oprettes i. Så slipper vi for et flag og det bliver nemmere at genbruge